Update various BN/NAT articles (#11368)
ppy/osu-wiki: wiki/People/Nomination_Assessment_Team/Evaluations/en.md

The duration between evaluations is automatically scheduled by the [BN website](https://bn.mappersguild.com/). However, if there is a substantial cause for concern about a BN's performance at a sooner date, NAT members can also manually schedule an evaluation to happen sooner than normal.

> A Nominator is able to see the current progress of their evaluation via a progress bar in their BN website profile.

Reviewer: is there a particular reason Nominator is used here instead of Beatmap Nominator? if its including NAT members then i think it should be clarified, nominators is a confusing term to encapsulate both BNs and NATs.

Author: ```suggestion
A BN is able to see the current progress of their evaluation via a progress bar in their BN website profile.
```

Once a consensus for the resignation terms has been reached, the NAT will send the BN an announcement message via the [Nomination Assessment Team bot](https://osu.ppy.sh/users/6616586) informing them of their status and when they can reapply to become a BN if they so choose.

#### Good terms

Beatmap Nominators who have not been recently warned or been on probation, and were not about to be warned for issues, will usually resign on good terms.

The good-term status allows former Nominators to fast-track their way back to being a full BN if they choose to apply again within 1 year of their resignation. The resignation will turn to standard terms if it is older than 1 year.

#### Standard terms

A standard-term resignation is issued for nominators who have been warned recently, are on probation, or were about to be warned for issues on their upcoming current BN evaluation.

Nominators who resign on standard terms will have to reapply to become a BN after a 60-day cooldown if they wish to return to the group, and they will have to start from probation if they pass their reapplication.

### Activity

> Whenever a Beatmap Nominator's evaluation is created, their activity will be checked automatically by the BN website. If it's failing to meet the [minimum requirements](/wiki/People/Beatmap_Nominators/Rules#activity), then the evaluation will be automatically moved to the group phase, have its consensus set to an activity warning or a removal, depending on the activity. Then, a member of the NAT will sanity check said evaluation, and send it if there are no issues.

Reviewer 2: ```suggestion
Whenever a Beatmap Nominator's evaluation is created, their activity will be checked automatically by the BN website. If it fails to meet the [minimum requirements](/wiki/People/Beatmap_Nominators/Rules#activity), then the evaluation will be automatically moved to the group phase and have its consensus set to an activity warning or a removal, depending on the activity. Then, a member of the NAT will sanity check said evaluation, and send it if there are no issues.
```

> If a Beatmap Nominator is returning from a good-term resignation and they haven't reached their minimum activity within the first 30 days, an evaluation following the same procedure above will be created, with its consensus being set to a removal.

Reviewer 2: ```suggestion
If a Beatmap Nominator is returning from a good-term resignation and they have not reached their minimum activity within the first 30 days, a new evaluation will be created following the same procedure as above, with the consensus set to a removal.
```
